charts/loki-stack/values.yaml

```diff
@@ -20,14 +20,14 @@ loki:
     jsonData: {}
     uid: ""
 
-
 promtail:
   enabled: true
   config:
     logLevel: info
     serverPort: 3101
     clients:
-      - url: http://{{ .Release.Name }}:3100/loki/api/v1/push
+      - url: http://{{ .Release.Name }}-loki.{{ .Release.Namespace }}:{{ .Values.loki_svc_port }}/loki/api/v1/push
+      # - url: http://{{ include "loki.serviceName" .Subcharts.loki }}:{{ include "loki.service.port" .Subcharts.loki }}/loki/api/v1/push
```

Comment:
imho, you should remove this comment :)

Reply:
@nervo I don't see it in the latest diff, can you take a look again at the latest diff?

Reply:
@CrazyByDefault i guess we have access to the same diff :) https://github.com/grafana/helm-charts/pull/1842/files (and no, i don't see it either)